test/framework/k8s_cluster.go

```diff
@@ -594,6 +594,44 @@ func (c *K8sCluster) UpgradeKuma(mode string, fs ...DeployOptionsFunc) error {
 	return nil
 }
 
+func (c *K8sCluster) ShutdownCP() error {
```

Comment:
Nit: suggest calling this pair of function `StopControlPlane` and `RestartControlPlane`. Not a blocker, up to you :)